src/explore-education-statistics-common/src/modules/table-tool/components/utils/downloadTableOdsFile.ts

```diff
@@ -120,7 +122,26 @@ export function appendFootnotes(
       ...footnotes.map((footnote, index) => [
         {
           t: 's',
-          v: `(${index + 1}) ${footnote.label}`,
+          // Footnotes can have links in so we want to remove the
+          // tags and append the urls after the link text.
+          v: `(${index + 1}) ${sanitizeHtml(footnote.label, {
+            allowedTags: [],
+            transformTags: {
+              a: (tagName, attribs) => {
+                linkHref = attribs.href;
+                return {
+                  tagName,
+                  attribs,
+                };
```

Comment:
What an awkward API. It would have been nice of them to provide the text in the callback params.

Reply:
yep!